### PR TITLE
Adding Sysdig Cloud example

### PR DESCRIPTION
**Pull request by 03cranec**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/29086)

### CODE REVIEW

**Comment by bgrant0607** [comment]
ok to test

**Comment by bgrant0607** [comment]
cc @kubernetes/examples @vishh

File: examples/sysdig-cloud/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+[Sysdig Cloud](http://www.sysdig.com/) is the first and only monitoring, alerting, and troubleshooting solution designed from the ground up to provide unprecedented visibility into containerized infrastructures.
```
Comment (jeffmendoza) [inline examples/sysdig-cloud/README.md]: Can this be worded a bit more neutral and less buzzwordy.

File: examples/sysdig-cloud/sysdig-daemonset.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+kind: DaemonSet                     
+metadata:
+  name: sysdig-agent
+  labels:
+    app: sysdig-agent
+spec:
+  template:
+    metadata:
+      labels:
+        name: sysdig-agent
```
Comment (jeffmendoza) [inline examples/sysdig-cloud/sysdig-daemonset.yaml]: labels of `name:` are discouraged.

File: examples/sysdig-cloud/sysdig-rc.yaml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+metadata:
+  name: sysdig-agent
+  labels:
+    app: sysdig-agent
+spec:
+  replicas: 100             #REQUIRED - replace with the maximum number of slave nodes in the cluster
+  template:
+    metadata:
+      labels:
+        name: sysdig-agent
```
Comment (jeffmendoza) [inline examples/sysdig-cloud/sysdig-rc.yaml]: same

File: examples/sysdig-cloud/sysdig-rc.yaml [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+          readOnly: true
+        - mountPath: /host/boot
+          name: boot-vol
+          readOnly: true
+        - mountPath: /host/lib/modules
+          name: modules-vol
+          readOnly: true
+        - mountPath: /host/usr
+          name: usr-vol
+          readOnly: true
```
Comment (jeffmendoza) [inline examples/sysdig-cloud/sysdig-rc.yaml]: Missing trailing newline.

**Comment by jeffmendoza** [comment]
```
/go/src/k8s.io/kubernetes/examples/ is out of date. Please run hack/update-munge-docs.sh
FAILED   hack/make-rules/../../hack/verify-munge-docs.sh	5s
```

**Comment by 03cranec** [comment]
@jeffmendoza I'm still getting the same build error - any suggestions?

**Comment by jeffmendoza** [comment]
Does `hack/verify-munge-docs.sh` run cleanly for you?

**Comment by janetkuo** [comment]
Just run hack/update-munge-docs.sh and commit the files updated by the script

**Comment by 03cranec** [comment]
It seems to, yes.

**This is the output when I run it now:**
$ bash hack/update-munge-docs.sh
Go version: go version go1.6.3 darwin/amd64
+++ [0725 14:35:59] Building the toolchain targets:
    k8s.io/kubernetes/hack/cmd/teststale
+++ [0725 14:35:59] Building go targets for darwin/amd64:
    cmd/mungedocs
+++ [0725 14:36:00] Placing binaries

**And this was the output the first time:**

$ bash hack/update-munge-docs.sh
Go version: go version go1.6.3 darwin/amd64
+++ [0725 12:12:10] Building the toolchain targets:
    k8s.io/kubernetes/hack/cmd/teststale
+++ [0725 12:12:11] Building go targets for darwin/amd64:
    cmd/libs/go2idl/deepcopy-gen
+++ [0725 12:12:14] Placing binaries
I0725 12:12:18.005820   72846 execute.go:214] Processing package "policy", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/policy"
I0725 12:12:18.010651   72846 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/policy/zz_generated.deepcopy.go"
I0725 12:12:18.012994   72846 execute.go:214] Processing package "v1beta1", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/extensions/v1beta1"
I0725 12:12:18.038415   72846 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/extensions/v1beta1/zz_generated.deepcopy.go"
I0725 12:12:18.054166   72846 execute.go:214] Processing package "v1alpha1", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/rbac/v1alpha1"
I0725 12:12:18.058038   72846 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/rbac/v1alpha1/zz_generated.deepcopy.go"
I0725 12:12:18.062368   72846 execute.go:214] Processing package "v1", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/autoscaling/v1"
I0725 12:12:18.066073   72846 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/autoscaling/v1/zz_generated.deepcopy.go"
I0725 12:12:18.068218   72846 execute.go:214] Processing package "v1beta1", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/authentication.k8s.io/v1beta1"
I0725 12:12:18.069382   72846 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/authentication.k8s.io/v1beta1/zz_generated.deepcopy.go"
I0725 12:12:18.070495   72846 execute.go:214] Processing package "federation", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/federation/apis/federation"
I0725 12:12:18.072351   72846 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/federation/apis/federation/zz_generated.deepcopy.go"
I0725 12:12:18.074797   72846 execute.go:214] Processing package "v1alpha1", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/policy/v1alpha1"
I0725 12:12:18.076163   72846 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/policy/v1alpha1/zz_generated.deepcopy.go"
I0725 12:12:18.077962   72846 execute.go:214] Processing package "v1alpha1", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/apps/v1alpha1"
I0725 12:12:18.080049   72846 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/apps/v1alpha1/zz_generated.deepcopy.go"
I0725 12:12:18.081489   72846 execute.go:214] Processing package "authentication", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/authentication.k8s.io"
I0725 12:12:18.082664   72846 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/authentication.k8s.io/zz_generated.deepcopy.go"
I0725 12:12:18.083765   72846 execute.go:214] Processing package "v1beta1", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/authorization/v1beta1"
I0725 12:12:18.086303   72846 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/authorization/v1beta1/zz_generated.deepcopy.go"
I0725 12:12:18.088125   72846 execute.go:214] Processing package "authorization", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/authorization"
I0725 12:12:18.091152   72846 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/authorization/zz_generated.deepcopy.go"
I0725 12:12:18.094420   72846 execute.go:214] Processing package "rbac", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/rbac"
I0725 12:12:18.104747   72846 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/rbac/zz_generated.deepcopy.go"
I0725 12:12:18.110352   72846 execute.go:214] Processing package "v2alpha1", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/batch/v2alpha1"
I0725 12:12:18.121707   72846 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/batch/v2alpha1/zz_generated.deepcopy.go"
I0725 12:12:18.127873   72846 execute.go:214] Processing package "certificates", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/certificates"
I0725 12:12:18.131250   72846 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/certificates/zz_generated.deepcopy.go"
I0725 12:12:18.136570   72846 execute.go:214] Processing package "autoscaling", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/autoscaling"
I0725 12:12:18.141361   72846 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/autoscaling/zz_generated.deepcopy.go"
I0725 12:12:18.145362   72846 execute.go:214] Processing package "apps", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/apps"
I0725 12:12:18.149604   72846 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/apps/zz_generated.deepcopy.go"
I0725 12:12:18.151963   72846 execute.go:214] Processing package "v1beta1", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/federation/apis/federation/v1beta1"
I0725 12:12:18.154877   72846 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/federation/apis/federation/v1beta1/zz_generated.deepcopy.go"
I0725 12:12:18.158105   72846 execute.go:214] Processing package "batch", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/batch"
I0725 12:12:18.167407   72846 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/batch/zz_generated.deepcopy.go"
I0725 12:12:18.171912   72846 execute.go:214] Processing package "v1alpha1", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/componentconfig/v1alpha1"
I0725 12:12:18.193016   72846 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/componentconfig/v1alpha1/zz_generated.deepcopy.go"
I0725 12:12:18.201837   72846 execute.go:214] Processing package "v1", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/api/v1"
I0725 12:12:18.270552   72846 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/api/v1/zz_generated.deepcopy.go"
I0725 12:12:18.320044   72846 execute.go:214] Processing package "componentconfig", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/componentconfig"
I0725 12:12:18.327786   72846 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/componentconfig/zz_generated.deepcopy.go"
I0725 12:12:18.332192   72846 execute.go:214] Processing package "api", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/api"
I0725 12:12:18.391920   72846 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/api/zz_generated.deepcopy.go"
I0725 12:12:18.447169   72846 execute.go:214] Processing package "v1", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/batch/v1"
I0725 12:12:18.450355   72846 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/batch/v1/zz_generated.deepcopy.go"
I0725 12:12:18.452396   72846 execute.go:214] Processing package "extensions", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/extensions"
I0725 12:12:18.478534   72846 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/extensions/zz_generated.deepcopy.go"
I0725 12:12:18.497679   72846 execute.go:214] Processing package "unversioned", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/api/unversioned"
I0725 12:12:18.502596   72846 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/api/unversioned/zz_generated.deepcopy.go"
I0725 12:12:18.505127   72846 execute.go:214] Processing package "v1alpha1", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/certificates/v1alpha1"
I0725 12:12:18.507518   72846 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/certificates/v1alpha1/zz_generated.deepcopy.go"
I0725 12:12:18.513607   72846 execute.go:214] Processing package "runtime", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/runtime"
I0725 12:12:18.516384   72846 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/runtime/zz_generated.deepcopy.go"
I0725 12:12:18.517283   72846 main.go:78] Completed successfully.
Go version: go version go1.6.3 darwin/amd64
+++ [0725 12:12:18] Building the toolchain targets:
    k8s.io/kubernetes/hack/cmd/teststale
+++ [0725 12:12:18] Building go targets for darwin/amd64:
    cmd/libs/go2idl/conversion-gen
+++ [0725 12:12:20] Placing binaries
I0725 12:12:23.098850   72971 execute.go:214] Processing package "v1", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/autoscaling/v1"
I0725 12:12:23.104046   72971 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/autoscaling/v1/zz_generated.conversion.go"
I0725 12:12:23.107821   72971 execute.go:214] Processing package "v1alpha1", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/rbac/v1alpha1"
I0725 12:12:23.116829   72971 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/rbac/v1alpha1/zz_generated.conversion.go"
I0725 12:12:23.120988   72971 execute.go:214] Processing package "v1", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/api/v1"
I0725 12:12:23.239424   72971 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/api/v1/zz_generated.conversion.go"
I0725 12:12:23.292789   72971 execute.go:214] Processing package "v2alpha1", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/batch/v2alpha1"
I0725 12:12:23.303899   72971 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/batch/v2alpha1/zz_generated.conversion.go"
I0725 12:12:23.308859   72971 execute.go:214] Processing package "v1alpha1", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/certificates/v1alpha1"
I0725 12:12:23.312843   72971 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/certificates/v1alpha1/zz_generated.conversion.go"
I0725 12:12:23.314797   72971 execute.go:214] Processing package "v1beta1", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/authorization/v1beta1"
I0725 12:12:23.320312   72971 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/authorization/v1beta1/zz_generated.conversion.go"
I0725 12:12:23.322803   72971 execute.go:214] Processing package "v1alpha1", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/apps/v1alpha1"
E0725 12:12:23.325561   72971 conversion.go:592] Warning: could not generate autoConvert functions for k8s.io/kubernetes/pkg/apis/apps/v1alpha1.PetSetSpec <-> k8s.io/kubernetes/pkg/apis/apps.PetSetSpec
I0725 12:12:23.325992   72971 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/apps/v1alpha1/zz_generated.conversion.go"
I0725 12:12:23.327203   72971 execute.go:214] Processing package "v1", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/batch/v1"
I0725 12:12:23.333478   72971 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/batch/v1/zz_generated.conversion.go"
I0725 12:12:23.336033   72971 execute.go:214] Processing package "v1alpha1", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/componentconfig/v1alpha1"
I0725 12:12:23.345263   72971 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/componentconfig/v1alpha1/zz_generated.conversion.go"
I0725 12:12:23.348812   72971 execute.go:214] Processing package "v1beta1", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/federation/apis/federation/v1beta1"
I0725 12:12:23.353858   72971 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/federation/apis/federation/v1beta1/zz_generated.conversion.go"
I0725 12:12:23.356264   72971 execute.go:214] Processing package "v1alpha1", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/policy/v1alpha1"
I0725 12:12:23.362006   72971 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/policy/v1alpha1/zz_generated.conversion.go"
I0725 12:12:23.363580   72971 execute.go:214] Processing package "v1beta1", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/authentication.k8s.io/v1beta1"
I0725 12:12:23.366179   72971 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/authentication.k8s.io/v1beta1/zz_generated.conversion.go"
I0725 12:12:23.367924   72971 execute.go:214] Processing package "v1beta1", disk location "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/extensions/v1beta1"
E0725 12:12:23.388618   72971 conversion.go:592] Warning: could not generate autoConvert functions for k8s.io/kubernetes/pkg/apis/extensions/v1beta1.HorizontalPodAutoscalerSpec <-> k8s.io/kubernetes/pkg/apis/autoscaling.HorizontalPodAutoscalerSpec
E0725 12:12:23.397473   72971 conversion.go:592] Warning: could not generate autoConvert functions for k8s.io/kubernetes/pkg/apis/extensions/v1beta1.JobSpec <-> k8s.io/kubernetes/pkg/apis/batch.JobSpec
E0725 12:12:23.413500   72971 conversion.go:592] Warning: could not generate autoConvert functions for k8s.io/kubernetes/pkg/apis/extensions/v1beta1.RollingUpdateDeployment <-> k8s.io/kubernetes/pkg/apis/extensions.RollingUpdateDeployment
E0725 12:12:23.415770   72971 conversion.go:592] Warning: could not generate autoConvert functions for k8s.io/kubernetes/pkg/apis/extensions/v1beta1.ScaleStatus <-> k8s.io/kubernetes/pkg/apis/extensions.ScaleStatus
I0725 12:12:23.419291   72971 execute.go:67] Assembling file "/Users/chris/Code/03cranec-kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apis/extensions/v1beta1/zz_generated.conversion.go"
I0725 12:12:23.446503   72971 main.go:73] Completed successfully.
Go version: go version go1.6.3 darwin/amd64
+++ [0725 12:12:23] Building the toolchain targets:
    k8s.io/kubernetes/hack/cmd/teststale
+++ [0725 12:12:23] Building go targets for darwin/amd64:
    cmd/mungedocs
+++ [0725 12:12:24] Placing binaries
/Users/chris/Code/03cranec-kubernetes/examples/sysdig-cloud/README.md
----
remove-whitespace:
contents were modified

unversioned-warning:
contents were modified

analytics:
contents were modified

**Comment by 03cranec** [comment]
@janetkuo that is what I did already. sorry if i'm missing something.

**Comment by jeffmendoza** [comment]
@03cranec Weird. I checked out your branch and got a failure, munge gives:

```
diff --git a/examples/sysdig-cloud/README.md b/examples/sysdig-cloud/README.md
index a4e5a96..2332945 100644
--- a/examples/sysdig-cloud/README.md
+++ b/examples/sysdig-cloud/README.md
@@ -2,15 +2,15 @@
 
 <!-- BEGIN STRIP_FOR_RELEASE -->
 
-<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+<img src="http://kubernetes.io/kubernetes/img/warning.png" alt="WARNING"
      width="25" height="25">
-<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+<img src="http://kubernetes.io/kubernetes/img/warning.png" alt="WARNING"
      width="25" height="25">
-<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+<img src="http://kubernetes.io/kubernetes/img/warning.png" alt="WARNING"
      width="25" height="25">
-<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+<img src="http://kubernetes.io/kubernetes/img/warning.png" alt="WARNING"
      width="25" height="25">
-<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+<img src="http://kubernetes.io/kubernetes/img/warning.png" alt="WARNING"
      width="25" height="25">
 
 <h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
```

**Comment by bgrant0607** [comment]
@jeffmendoza Eventually it would be nice to group all the monitoring examples under a `monitoring` subdirectory.

**Comment by 03cranec** [comment]
ah, think I got it. i needed to pull from kubernetes/master .. i was running an outdated version of the hack script. my bad. let's see if this lasted commit works.

**Comment by jeffmendoza** [comment]
LGTM

**Comment by jeffmendoza** [comment]
@bgrant0607 Sounds good, I'll put a list in examples/README.md of the current category directories.

**Comment by jeffmendoza** [comment]
@janetkuo can you help with a LGTM label? thanks

**Comment by erictune** [comment]
Fine to put this in `examples/` right now.

In the future, I want the `examples/` directory to have fewer examples, and examples that focus on teaching Kubernetes concepts.  This PR is more about quickly getting SysDig quickly running, which is awesome.  I foresee moving this into a chart once we have a better workflow for chart contributions and releases.

**Comment by jeffmendoza** [comment]
>  I foresee moving this into a chart once we have a better workflow for chart contributions and releases.

Agree

**Comment by 03cranec** [comment]
sounds good. thanks all. anything else you need from me? @janetkuo?

**Pull request merged**